assets/base/i18n/es.json

```diff
@@ -332,20 +346,27 @@
   "mobile.open_dm.error": "No pudimos abrir el canal de mensajes directos con {displayName}. Por favor revisa tu conexión e intenta de nuevo.",
   "mobile.open_gm.error": "No pudimos abrir el canal del grupo con esos usuarios. Por favor revisa tu conexión e intenta de nuevo.",
   "mobile.open_unknown_channel.error": "No se pudo unir al canal. Por favor elimina el cache e intenta de nuevo.",
+  "mobile.permission_denied_dismiss": "Don't Allow",
+  "mobile.permission_denied_retry": "Configuración",
```

Comment:
Not for this PR, but looks like the `mobile.permission_denied_retry` id for translation is translated to `Settings` everywhere, what makes me think, or the id makes no sense, or the implementation makes no sense. If we want to show `Settings` there, probably the the id should be something like `mobile.settings_on_permission_denied` or something like that (Maybe I'm wrong because I'm not seeing the context whole picture here).

Reply:
yeah perhaps the id can be changed to `mobile.permission_denied_go_to_settings` or something like that.

To give you context, this happens when the user permission is required to access photos, camera, files, etc.. and the permission was already denied, so we need them to go the the Phone Settings and change it manually.